src/styles.js

```diff
@@ -1,4 +1,5 @@
 import { createGlobalStyle } from 'styled-components';
+import styled from 'styled-components';
```

Comment:
Estas 2 líneas pueden ser una, xq los 2 se importan de la misma lib
```suggestion
import styled, { createGlobalStyle } from 'styled-components';
```